app/unit-tests/src/org/commcare/android/mocks/ModernHttpRequesterMock.java

```diff
@@ -79,11 +101,19 @@ protected HttpURLConnection setupConnection(URL builtUrl) throws IOException {
                 } catch (InvalidReferenceException ire) {
                     throw new IOException("No payload available at " + payloadReference);
                 }
-                return HttpURLConnectionMock.mockWithStream(builtUrl, responseCodeStack.remove(0), payloadStream);
+                return OkHTTPResponseMock.createResponse(responseCodeStack.remove(0),payloadStream);
             }
         }
     }
 
+    private URL buildUrlWithParams() throws MalformedURLException {
```

Comment:
Since the only usage of this method immediately calls `toString()` on the returned result, might as well make the return type `String` instead